[infra-001] test_very_long_content: 50KB content rejected by 8000-byte content size cap

## Discovered By

Integration test: `suites/test_edge_cases.py::test_very_long_content`

## Expected Behavior

50,000-byte (50KB) content should be accepted by `context_store` and return success.

## Actual Behavior

Server returns error: `Invalid parameter 'content': Content exceeds configured maximum of 8000 bytes (received 50000 bytes). Consider breaking this into multiple focused entries.`

## Root Cause

Fix `b709de06` (GH#561/#573) added a 8000-byte content size cap to `context_store` and `context_correct`. The test `test_very_long_content` predates this fix and sends 50KB, which now exceeds the cap.

## Server Logs

```
AssertionError: JSON-RPC error -32602: Invalid parameter 'content': Content exceeds configured maximum of 8000 bytes (received 50000 bytes).
```

## Reproduction

```bash
cd product/test/infra-001
python -m pytest suites/test_edge_cases.py::test_very_long_content -v
```

## Notes

Discovered during infra-001 integration testing for vnc-014. Not caused by vnc-014 changes. The test needs to be updated to either use content within the 8KB cap or to assert that the error is returned. The test was validating near-boundary behavior but the boundary changed.

--- a/packages/unimatrix/test/init.test.js
+++ b/packages/unimatrix/test/init.test.js
@@ -96,7 +96,9 @@ describe("writeMcpJson", () => {
     const content = JSON.parse(fs.readFileSync(mcpPath, "utf8"));
     assert.strictEqual(content.mcpServers.unimatrix.command, BINARY);
     assert.deepStrictEqual(content.mcpServers.unimatrix.args, []);
-    assert.deepStrictEqual(content.mcpServers.unimatrix.env, {});
+    assert.deepStrictEqual(content.mcpServers.unimatrix.env, {
+      LD_LIBRARY_PATH: path.dirname(BINARY),
+    });
 
     assert.ok(
       actions.some((a) => a.includes("Created .mcp.json")),

--- a/product/test/infra-001/suites/test_edge_cases.py
+++ b/product/test/infra-001/suites/test_edge_cases.py
@@ -228,10 +228,9 @@ def test_interleaved_store_and_search(server):
         server.context_search(f"interleaved entry {i}", k=2)
 
 
-@pytest.mark.xfail(reason="Pre-existing: GH#576 — content size cap of 8000 bytes (fix #561) now rejects 50KB content; test predates the cap")
 def test_very_long_content(server):
     """E-15: Near-boundary-length content."""
-    content = load_large_content(50000)  # 50KB
+    content = load_large_content(8000)  # 8000 bytes — content size cap boundary (passes)
     resp = server.context_store(
         content, "testing", "convention", agent_id="human"
     )

--- a/product/test/infra-001/suites/test_tools.py
+++ b/product/test/infra-001/suites/test_tools.py
@@ -1171,7 +1171,6 @@ def test_retrospective_json_explicit(server):
     assert "feature_cycle" in parsed, f"Expected feature_cycle in JSON, got keys: {list(parsed.keys())}"
 
 
-@pytest.mark.xfail(reason="Pre-existing: GH#575 — error message is 'Invalid parameter format: must be summary, markdown, or json' not 'Unknown format'")
 def test_retrospective_format_invalid(server):
     """T-R09 (vnc-011): Invalid format returns error with descriptive message."""
     features = ["col-833"]
@@ -1179,7 +1178,7 @@ def test_retrospective_format_invalid(server):
     _seed_observation_sql(db_path, features)
 
     resp = server.context_cycle_review(features[0], agent_id="human", format="xml", timeout=30.0)
-    assert_tool_error(resp, "Unknown format")
+    assert_tool_error(resp, "must be summary, markdown, or json")
 
 
 # === context_status observation extension (col-002) =======================

